### PR TITLE
Update cs5001.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/csharp/misc/cs5001.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+helpviewer_keywords:
   - "CS5001"
 ms.assetid: e1e26e75-84e0-47c7-be8a-3c4fd0d6f497
 ---
 # Compiler Error CS5001
 
-Program 'program' does not contain a static 'Main' method suitable for an entry point  
-  
-This error occurs when no static `Main` method with a correct signature is found in the code that produces an executable file. This error also occurs if the entry point function, `Main`, is defined with the wrong case, such as lower-case `main`. For information about the rules that apply to the `Main` method, see [Main() and Command-Line Arguments](../programming-guide/main-and-command-args/index.md).
+Program does not contain a static 'Main' method suitable for an entry point
```
Comment (mairaw) [inline docs/csharp/misc/cs5001.md]: I think the error message contains the name of the program in the middle of the sentence, doesn't it? Let me double check.

If it does, we should keep the original version.

File: docs/csharp/misc/cs5001.md [hunk 2]
```diff
@@ -22,15 +22,16 @@ The `Main` method is only required when compiling an executable file, that is, w
 - WPF application
 - Windows Forms application
 
-## Example  
- The following example generates CS5001:  
+## Example
+
+ The following example generates CS5001:
```
Comment (mairaw) [inline docs/csharp/misc/cs5001.md]:
```suggestion
The following example generates CS5001:
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Left a few comments.

File: docs/csharp/misc/cs5001.md (hunk 1 above)
Comment (Youssef1313) [reply docs/csharp/misc/cs5001.md]: I didn't generate the error by myself, but have checked Roslyn repo and didn't find it contains the program name.

Comment (Youssef1313) [reply docs/csharp/misc/cs5001.md]: https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/CSharpResources.resx#L3779

Comment (mairaw) [reply docs/csharp/misc/cs5001.md]: Yep. I've created a quick test and also confirmed that. Thanks!

**Review by mairaw — APPROVED** [review]
:shipit:

File: docs/csharp/misc/cs5001.md (hunk 1 above)
Comment (Youssef1313) [reply docs/csharp/misc/cs5001.md]: I've tried in Visual Studio and program name isn't included.

image

However, In some older versions of C# compiler, (not sure if it's even before Roslyn or not), the program name **was** included.

**Pull request merged**